stanford_image.ds.inc

```diff
@@ -126,6 +126,15 @@ function stanford_image_ds_view_modes_info() {
 
   $ds_view_mode = new stdClass();
   $ds_view_mode->api_version = 1;
+  $ds_view_mode->view_mode = 'stanford_full_width_banner';
+  $ds_view_mode->label = 'Banner Full Width';
```

Comment:
@cynmij this is called "Full Width Banner" everywhere else. Do you want the view mode to be called "Banner Full Width"?